server/settings/main.py

```diff
@@ -276,31 +277,38 @@ class CoreSettings(BaseSettingsModel):
     disk_mapping: DiskMappingModel = SettingsField(
         default_factory=DiskMappingModel,
         title="Disk mapping",
+        description="Map virtual disk(s) on artist's machine for AYON when running",
```

Comment:
This doesn't map _virtual_ disks, right? It maps / mounts a folder to another path. Virtual disks to me sound like a very different thing.
```suggestion
        description="Map a (network) drive to artist's machine for AYON when running",
```

See:
- https://support.microsoft.com/en-us/windows/map-a-network-drive-in-windows-29ce55d1-34e3-a7e2-4801-131475f9557d
- https://askubuntu.com/questions/46183/how-to-map-a-network-drive

But maybe I'm misunderstanding the feature. Also here, it'd be great to have in the description when to use it.

Reply:
I had to refer to our user doc for this one (as I wasnt so sure) , so it must be wrong too...